hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +27,297 @@ toc::[]
+
+** *Fix*: Instead of relying on the condition state, NMC can rely on the `Status.NodeInfo.BootID` field. This field is set by kubelet based on the /proc/sys/kernel/random/boot_id file of the server node, so it is updated after each reboot.
+
+** *Result:* The more accurate timestamps enable the Kernel Module Management (KMM) Operator to reload the kernel module after the node reboot sequence.
+
+// MGMT-20248
+* Filtering out node heartbeats events for the NMC controller.
+
+** *Cause*: The Node Machine Configuration (NMC) controller gets spammed with events from node heartbeats. The node heartbeats are there to let the Kubernetes API server know that the node is still connected and functional.
+
+** *Consequence*: The spamming causes a constant reconciliation even when no Module, and therefore no NMC, are applied to the cluster.
+
+** *Fix*: The NMC controller will now filter the node's heartbeat from its reconciliation loop.
+
+** *Result:* The NMC controller only gets real events and filters out node heartbeats.
+
+// MGMT-20286
+* NMC status contains toleration values, even though there are no tolerations in the NMC.spec or in the Module. 
+
+** *Cause*: The Node Machine Configuration (NMC) status contains toleration values, even though there are no tolerations in the NMC.spec or in the Module.
+
+** *Consequence*: Tolerations other than Kernel Module Management-specific tolerations can appear in the status.
+
+** *Fix*: The NMC status now gets its toleration from a dedicated annotation rather than from the worker pod.
+
+** *Result:* The NMC status only contains the Module's tolerations.
+
+// MGMT-20725
+* The KMM Operator version 2.4 fails to start properly and cannot list the resource `\modulebuildsignconfigs\`. 
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, when the Operator is installed using Red Hat Konflux it does not start properly as the log files contain errors.
+
+** *Consequence*: The KMM Operator does not work as expected.
+
+** *Fix*: The Cluster Service Version (CSV) file was updated to list the resources `\modulebuildsignconfigs\` and `moduleimagesconfig`.
+
+** *Result:* The KMM Operator works as expected.
+
+// MGMT-20752
+* Konflux build does not include version and git commit in the operator logs.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, when the Operator was built using Communications Platform as a Service (CPaas), the build included the Operator version and git commit in the log  files. However, with Red Hat Konflux these details are not included in the log files.
+
+** *Consequence*: Important information is missing from the log files.
+
+** *Fix*: Some adjustments were made in Konflux.
+
+** *Result:* The KMM Operator build now includes the Operator version and git commit in the log files.
+
+// MGMT-20754
```

Comment:
Duplicate entry of teh same issue [here](https://github.com/openshift/openshift-docs/pull/95248/files#diff-a508e43343db975c5b8b7e3b4f30ef90441b9420f08df906454a6d85aa693514R144)